TVShows/Modules/Login/LoginViewController.swift

```diff
@@ -58,3 +58,104 @@ private extension LoginViewController {
     }
 
 }
+
+extension LoginViewController: ConstructViewsInterface {
+    func buildViews() {
+        addViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    func addViews() {
+        view.addSubview(scrollView)
+
+        scrollView.addSubview(contentView)
+
+        contentView.addSubview(contentStackView)
+
+        contentStackView.addArrangedSubview(logoImageView)
+
+        contentStackView.addArrangedSubview(loginLabel)
+
+        contentStackView.addArrangedSubview(subtitleLabel)
+
+        contentStackView.addArrangedSubview(emailInputView)
+
+        contentStackView.addArrangedSubview(passwordInputView)
+
+        contentStackView.addArrangedSubview(rememberMeButton)
+
+        contentStackView.addArrangedSubview(loginButton)
+
+        contentStackView.addArrangedSubview(registerButton)
+
+        view.addGestureRecognizer(
```

Comment:
Too much empty new lines 😂 No need for this
```suggestion
        view.addSubview(scrollView)
        scrollView.addSubview(contentView)
        contentView.addSubview(contentStackView)
        contentStackView.addArrangedSubview(logoImageView)
        contentStackView.addArrangedSubview(loginLabel)
        contentStackView.addArrangedSubview(subtitleLabel)
        contentStackView.addArrangedSubview(emailInputView)
        contentStackView.addArrangedSubview(passwordInputView)
        contentStackView.addArrangedSubview(rememberMeButton)
        contentStackView.addArrangedSubview(loginButton)
        contentStackView.addArrangedSubview(registerButton)

        view.addGestureRecognizer(
```